analytical_engine/java/grape-jdk/src/main/java/com/alibaba/graphscope/ds/GrapeNbr.java

```diff
@@ -114,6 +103,6 @@
          * @param edata edge data.
          * @return Nbr instance.
          */
-        Nbr<VID, EDATA> create(VID lid, @CXXReference EDATA edata);
+        GrapeNbr<VID_T, EDATA_T> create(VID_T lid, @CXXReference EDATA_T edata);
```

Comment:
why made this renaming? Nbr -> GrapeNbr??

Reply:
We use `GrapeNbr` to map `grape::Nbr`, and `ProjectedNbr` to map `gs::Nbr` in java correspondingly.

Reply:
OK~